roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -0,0 +1,49 @@
+# These tasks are run when aci_mode is set
+
+- name: check aci-gw container image
+  shell: docker inspect contiv/aci-gw
+  register: docker_aci_inspect_result
+  ignore_errors: yes
```

Comment:
Yes, this task needs to have run_once. Will make the change.

But the reason I didn't include it in the include level was that we might need contiv-compose on all nodes. @jojimt @jainvipin: Let me know if this is not a requirement. I can then move the run_once variable as necessary.